transformer/cloudfoundrytransformer.go

```diff
@@ -301,13 +359,34 @@ func flattenVariable(prefix string, credential interface{}) []core.EnvVar {
 	return credentialList
 }
 
+// createDbEnvironmentVariable creates a DATABASE_URL from the JSON value for VCAP_SERVICES
+func createDbEnvironmentVariable(vcapService string, serviceName string) (core.EnvVar, error) {
+	var serviceInstanceMap map[string][]artifacts.VCAPService
+	err := json.Unmarshal([]byte(vcapService), &serviceInstanceMap)
+	if err != nil {
+		return core.EnvVar{},
+			fmt.Errorf("Could not unmarshal the service map instance (%s) in VCAP_SERVICES during CF flattening: %s", vcapService, err)
+	}
+	for _, serviceInstances := range serviceInstanceMap {
+		for _, serviceInstance := range serviceInstances {
+			if serviceInstance.ServiceName == serviceName {
+				if uriValue, ok := serviceInstance.ServiceCredentials["uri"].(string); ok {
+					return core.EnvVar{Name: "DATABASE_URL", Value: uriValue}, nil
+				}
+				return core.EnvVar{}, fmt.Errorf("uri field is not available in service credential for the service VCAP_SERVICES")
+			}
+		}
+	}
+	return core.EnvVar{}, fmt.Errorf("service %s not available in VCAP_SERVICES", serviceName)
+}
```

Comment:
Do we need a separate function for this? why not do it within flattenVcapServiceVariables? We are already casting it into a vcap object there and also iterating over the services there.

Reply:
done